travis_ci_flake8.cfg

```diff
@@ -1,4 +1,12 @@
 [flake8]
+extend-exclude =
```

Comment:
I'm not completely sure if this change makes sense.

**pro**
- number of files looked at is reduced (=> faster flake8?)
- when introducing new rules, fewer violations have to be cleaned up at once

**contra**
- the list of ignored directories has to be maintained
- violations may slip through if other projects have no automated check (e.g. [senaite.jsonapi](https://github.com/senaite/senaite.jsonapi/pull/48#discussion_r933829253))

Reply:
maybe it makes more sense to explicitly only define the path to `senaite.core` here:
https://github.com/senaite/senaite.core/blob/2.x/.travis.yml#L23
e.g. `src/senaite.core`

Reply:
yep, that seems also a good approach—I've adjusted my changes here but also included `bika.lims` just to make sure we don't introduce new technical debt there that needs to be cleaned up during the migration.